regression/regdb.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python3
+
+from testing_tool import (TestCase, apply_transform_over_tests, FAIL_MODES)
+
+import sys
+import os.path
+import getopt
+import shlex
+
+SOLVERS = [
+    'smtlib',
+    'z3',
+    'boolector',
+    'cvc',
+    'mathsat',
+    'yices',
+    'bitwuzla'
+]
+
+EXTENSIONS = {
+    'c'     : {'c', 'i'},
+    'cxx'   : {'cc', 'cpp'},
+    'cuda'  : {'cuda'},
+    'py'    : {'py'},
+    'sol'   : {'solast'},
+    'jimple': {'jimple'},
```

Comment:
We also have .goto

Reply:
It's under the `--binary` option as we don't have it by file name extension.